src/RBTree.mo

```diff
@@ -186,16 +423,26 @@ module {
     }
   };
 
-  func height<X, Y>(t : Tree<X, Y>) : Nat {
-    switch t {
-      case (#leaf) { 0 };
-      case (#node(_, l, _, r)) {
-        Nat.max(height(l), height(r)) + 1
-      }
-    }
-  };
-
-  /// The size of the tree as the number of key-value entries.
+  /// Determine the size of the tree as the number of key-value entries.
+  ///
+  /// Example:
+  /// ```motoko
+  /// import RBTree "mo:base/RBTree";
+  /// import Nat "mo:base/Nat";
+  ///
+  /// let tree = RBTree.RBTree<Nat, Text>(Nat.compare);
+  /// tree.put(1, "one");
+  /// tree.put(2, "two");
+  /// tree.put(3, "three");
+  ///
+  /// RBTree.size(tree.share()) // 3 entries
+  /// ```
+  ///
+  /// Runtime: `O(log(n))`.
+  /// Space: `O(1)` retained memory plus garbage, see the note below.
```

Comment:
Similar to the comment about `iter`. I get the feeling this should be `O(n)` runtime and `O(log(n))` space, except this time the space usage is from the call stack.

Reply:
Thanks a lot. I fixed this.